src/pages/[platform]/reference/permissions-boundary/index.mdx

```diff
@@ -29,7 +29,7 @@ export function getStaticProps(context) {
 }
 
 
-To set the maximum permissions that can be granted to IAM Roles created by Amplify, configure a [permissions boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) for the project. Then, Amplify-generated IAM roles can perform only the actions that are allowed by both the roles’ policies and permissions boundary. You can configure a different permissions boundary for each environment. For example, this enables you to deny a staging or dev environment access to a production environment's resources.
+To set the maximum permissions that can be granted to IAM Roles created by Amplify, configure a [permissions boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) for the project. Then, Amplify-generated IAM roles can perform only the actions that are allowed by both the roles’ policies and permissions boundary. 
```

Comment:
> for the project
This is really for the "AWS environment" which in CDK parlance is the entire Account + Region

Reply:
https://docs.aws.amazon.com/cdk/v2/guide/environments.html